tensorboard/plugins/custom_scalar/tf_custom_scalar_dashboard/tf-custom-scalar-margin-chart-card.html

```diff
@@ -561,18 +633,21 @@ <h1>[[_titleDisplayString]]</h1>
         return escapedTags.join('|');
       },
       _escapeRegexCharacters(stringValue) {
-        return stringValue.replace(
-            /[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&');
+        return stringValue.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
       },
       _getToggleMatchesIcon(matchesListOpened) {
         return matchesListOpened ? 'expand-less' : 'expand-more';
       },
       _toggleMatchesOpen() {
         this.set('_matchesListOpened', !this._matchesListOpened);
       },
-      _computeTitleDisplayString(title) {
-        // If no title is provided, use a placeholder string.
-        return title || 'untitled';
+      _computeTitleDisplayString(title, tagFilter) {
+        // If no title is provided, use the tag filter, which is a combination
+        // of the tags for the value, lower, and upper fields.
```

Comment:
"... of each series."

Might be pretty cumbersome for a multi-margin-series plot.  Perhaps "untitled" would be better after all :/

Reply:
Done.

Hmm, yeah, I guess in the worst case, we can have a really long tag filter because this is a concatenation of tags. SG. Using untitled.